pyiron/atomistics/structure/atoms.py

```diff
@@ -2367,6 +2368,42 @@ def get_symmetry(
                 angle_tolerance=angle_tolerance,
             )
 
+    def symmetrize_vector(
+        self, vector, force_update=False, use_magmoms=False, use_elements=True, symprec=1e-5, angle_tolerance=-1.0
+    ):
+        """
+        symmetrization of natom x 3 vector according to box symmetries
+
+        args:
+            vector (ndarray/list): natom x 3 array to symmetrize
+            force_update (bool): whether to update the symmetry info
+            use_magmoms: cf. get_symmetry
+            use_elements: cf. get_symmetry
+            symprec: cf. get_symmetry
+            angle_tolerance: cf. get_symmetry
```

Comment:
Are they relative tolerances or in units of Angstrom/radians?

Reply:
I don't think it's something that should be specified in this function, because it's just passed to `get_symmetry`